### PR TITLE
Fixed TruckersFM AutoDJ

### PR DESCRIPTION
**Pull request by NinjaLabs-Dev**
**Reason for fix**:
When on the website using PreMiD with AutoDJ, an error would be caused due to being no 'Until' time.

**Changes**: 
Added clause to stop the error

**Evidence of testing**:

Image
Image

### CODE REVIEW

File: websites/T/TruckersFM/presence.ts [hunk 1]
```diff
@@ -7,14 +7,14 @@ presence.on("UpdateData", () => {
     document.querySelector(".player-artist-text").textContent
   } - ${document.querySelector(".player-title-text").textContent} `;
   const dj = document.querySelector(".live-name").textContent;
-  const liveTill = document.querySelector(".live-time").textContent;
-  const liveTime = liveTill.slice(6);
+  const liveTill = document.querySelector(".live-time") ? document.querySelector(".live-time").textContent : false;
   const presenceData: PresenceData = {
```
Comment (Bas950) [inline websites/T/TruckersFM/presence.ts:11]: merge consts

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/T/TruckersFM/presence.ts [hunk 2]
```diff
@@ -7,14 +7,13 @@ presence.on("UpdateData", () => {
     document.querySelector(".player-artist-text").textContent
   } - ${document.querySelector(".player-title-text").textContent} `;
   const dj = document.querySelector(".live-name").textContent;
-  const liveTill = document.querySelector(".live-time").textContent;
-  const liveTime = liveTill.slice(6);
+  const liveTill = document.querySelector(".live-time") ? document.querySelector(".live-time").textContent.slice(6) : false;
   const presenceData: PresenceData = {
```
Comment (ririxidev) [inline websites/T/TruckersFM/presence.ts:11]: youd idn't merge consts yet lol

**Review by ririxidev — CHANGES_REQUESTED** [review]

File: websites/T/TruckersFM/presence.ts (hunk 2 above)
Comment (NinjaLabs-Dev) [reply websites/T/TruckersFM/presence.ts:11]: Huh?

I merged the liveTime and liveTill in https://github.com/PreMiD/Presences/pull/2867/commits/2fc6d6168ddb64e8ead3dd53f4b33bc8b0fbeb2c

Comment (NinjaLabs-Dev) [reply websites/T/TruckersFM/presence.ts:11]: Nevermind! I realised what you meant now aha

Comment (ririxidev) [reply websites/T/TruckersFM/presence.ts:11]: no look

Comment (ririxidev) [reply websites/T/TruckersFM/presence.ts:11]: image

Comment (ririxidev) [reply websites/T/TruckersFM/presence.ts:11]: these consts are not merged together...

**Review by ririxidev — CHANGES_REQUESTED** [review]

**Comment by NinjaLabs-Dev** [comment]
Are you wanting me to merge `dj` and `liveTill` together? Or are your speaking about `liveTill` and `stateText`

**Comment by ririxidev** [comment]
what? I want you to merge consts:
```javascript
const f = "f",
g = "gg",
hhh = "gsgsgs";
```

**Comment by NinjaLabs-Dev** [comment]
Ok, I've done what I think you want me to do. I'm still not 100% on what you're requesting

**Comment by ririxidev** [comment]
Not this bruh, read how to merge 2 or more variables... (Aka const / let)

**Comment by ririxidev** [comment]
You forgot to merge presencedata const too

**Comment by NinjaLabs-Dev** [comment]
There we go aha

**Comment by ririxidev** [comment]
Oh no... Uhhh wait you broke it. I didn't notice. Merging consts doesn't work like that. Ask on discord so maybe someone will help you, and let me know for now I'm closing.

**Pull request closed without merging**